test/Maven/PomStrategySpec.hs

```diff
@@ -26,6 +26,10 @@ spec = do
     it "should interpolate multiple properties" $ do
       interpolateProperties pom "${project.groupId}${project.artifactId}" `shouldBe` "MYGROUPMYARTIFACT"
 
+    it "should not infinitely recurse when interpolating a property that is interpolated to itself" $ do
+      let pom' = pom{pomProperties = Map.singleton "project.groupId" "${project.groupId"}
+      interpolateProperties pom' "${project.groupId}" `shouldBe` "OTHERGROUP"
+
```

Comment:
Two things need to be fixed here:

1) `"${project.groupId"` => `"${project.groupId}"`
2) the expected groupId should be `${project.groupId}`.

This should work:

```suggestion
it "should not infinitely recurse when interpolating a property that is interpolated to itself" $ do
      let pom' = pom{pomProperties = Map.singleton "project.groupId" "${project.groupId}"}
      interpolateProperties pom' "${project.groupId}" `shouldBe` "project.groupId"
```

```
cabal test unit-tests --test-option=--color --test-option="--match=should not infinitely recurse when interpolating a property that is interpolated to itself"
```

Reply:
Didn't know about `--test-option`, that makes it a lot easier to test locally. Thanks!